test/unit/spec/OktaSignIn_spec.js

```diff
@@ -222,4 +225,49 @@ function (Widget, Expect, Logger, $sandbox) {
       });
     });
   });
+
+
+  Expect.describe('OktaSignIn v2 initialization', function () {
+    let signIn;
+    beforeEach(function () {
+      spyOn(Logger, 'warn');
+      signIn = new Widget({
+        baseUrl: url,
+        stateToken: '01stateToken',
+        features: {
+          router: true
+        }
+      });
+      spyOn(signIn.authClient.tx, 'introspect').and.callFake(function () {
+        return Q(introspectResponse.response);
+      });
+    });
+
+    afterEach(function () {
+      signIn.remove();
+    });
+
+    function setupIntrospect () {
+      signIn.renderEl({ el: $sandbox });
+      return Expect.wait(() => {
+        return ($('.siw-main-body').length === 1);
+      });
+    }
+    Expect.describe('Introspects token and loads Identifier view for new pipeline', function () {
+      it('calls introspect API on page load', function () {
+        return setupIntrospect({
+          response: {
+            version: '1.0.0'
+          }
+        }).then(function () {
+          //TODO replace with idx js mock
+          expect(signIn.authClient.tx.introspect).toHaveBeenCalledWith({ stateToken: '01stateToken'});
+          expect($('.siw-main-body .okta-form-title').text()).toBe('Sign In');
```

Comment:
Can we use a dom helper like other places

Reply:
I'll make that change in my next PR when I will re-visit this test to update it to use idx-js ?